.github/workflows/release.yml

```diff
@@ -41,7 +41,12 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          
+      - name: Validate peer dependencies
+        if: steps.changesets.outputs.published == 'true'
+        run: |
+          pnpm dlx dependency-check --peer-deps-only
```

Comment:
what does `dependency-check` come from? which npm package?

Reply:
Here is the [link](https://github.com/dependency-check-team/dependency-check). I just noticed that it was archived yikes 😅. So i guess i will try using `knip`